main/states/susi_state_machine.py

```diff
@@ -80,6 +87,25 @@ def __init__(self, renderer=None):
         else:
             logger.warning("Susi has the wake button disabled")
             self.wake_button = None
+    
+    def server_checker(self):
+        response_one = None
+        test_params = {
+        'q': 'Hello',
```

Comment:
Please care the indentation (coding style)

Reply:
This has not been resolved